src/index.html

```diff
@@ -13,6 +13,47 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars stars__star"></div>
+      <div class="stars stars__star"></div>
+      <div class="stars stars__star"></div>
+      <div class="stars stars__star"></div>
+      <div class="stars stars__star"></div>
+    </div>
+    <div class="stars stars--1">
+      <div class="stars stars__star"></div>
+      <div class="stars stars__star"></div>
+      <div class="stars stars__star"></div>
+      <div class="stars stars__star"></div>
+      <div class="stars stars__star"></div>
+    </div>
+    <div class="stars stars--2">
+      <div class="stars stars__star"></div>
+      <div class="stars stars__star"></div>
+      <div class="stars stars__star"></div>
+      <div class="stars stars__star"></div>
+      <div class="stars stars__star"></div>
+    </div>
+    <div class="stars stars--3">
+      <div class="stars stars__star"></div>
+      <div class="stars stars__star"></div>
+      <div class="stars stars__star"></div>
+      <div class="stars stars__star"></div>
+      <div class="stars stars__star"></div>
+    </div>
+    <div class="stars stars--4">
+      <div class="stars stars__star"></div>
+      <div class="stars stars__star"></div>
+      <div class="stars stars__star"></div>
+      <div class="stars stars__star"></div>
+      <div class="stars stars__star"></div>
+    </div>
+    <div class="stars stars--5">
+      <div class="stars stars__star"></div>
+      <div class="stars stars__star"></div>
+      <div class="stars stars__star"></div>
+      <div class="stars stars__star"></div>
+      <div class="stars stars__star"></div>
```

Comment:
The class names `stars stars--0`, `stars stars__star`, etc., seem to be a mixture of BEM naming convention and something else. According to BEM, modifiers (`--modifier`) and elements (`__element`) should not be used together with the block name. Consider revising the class names to follow BEM properly. For example, the block could be `stars`, the element could be `stars__star`, and the modifier could be `stars--0`. Ensure that the class names represent the meaning of the content and not the styles or tag names.